charters/DataPortal/charter.md

```diff
@@ -0,0 +1,81 @@
+# Data Portal
+
+
+## Description
+The Data Portal provides the overall web-site for the HCA DCP with entry point from https://www.humancellatlas.org/. It provides the user experience for scientific and general users to upload, explore, search and run analyses on human genomic data. The Data Browser ("Explore") application provides a faceted search capability to find and download subsets of this data.
+
+## Objective
+Implementation of the Data Coordination Platform (DCP) website experience. Provide the access point for scientific and general users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis, providing file/bundle search on multiple clouds at PB scale, with strong public APIs.
```

Comment:
*Minor:*
* Even Amazon _spells out_ PB as either _petabyte scale_ or _petabyte-scale_. 
* What is meant by _string public APIs_?